Annotation/polar_golden_set.json

```diff
@@ -419,13 +456,13 @@
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "whether the Germans of the first century held land in private ownership",
                 "embedded clauses": []
             }
         ]
     },
     "118": {
-        "sentence": "Once the affidavit and proposed order have been received by the court, a judge will consider all of the information your provided and determine whether you qualify for poor person status.",
+        "sentence": "Once the affidavit and proposed order have been received by the court, a judge will consider all of the information you provided and determine whether you qualify for poor person status.",

```

Comment:
Are we supposed to be correcting typos in the sentences? I've been just leaving it as is. It might be better to test our tool against the imperfect/messy reality of corpus data?

Reply:
Same as the above reply from above comment on reference, I can leave the imperfections on the rest of the stuff - I'll just delete similar comments like this